gossipd/routing.c

```diff
@@ -1086,39 +1086,39 @@ static struct wireaddr *read_addresses(const tal_t *ctx, const u8 *ser)
 
 bool routing_add_node_announcement(struct routing_state *rstate, const u8 *msg TAKES)
 {
-       struct node *node;
-       secp256k1_ecdsa_signature signature;
-       u32 timestamp;
-       struct pubkey node_id;
-       u8 rgb_color[3];
-       u8 alias[32];
-       u8 *features, *addresses;
-       struct wireaddr *wireaddrs;
-       fromwire_node_announcement(tmpctx, msg,
-                                  &signature, &features, &timestamp,
-                                  &node_id, rgb_color, alias,
-                                  &addresses);
-
-       node = get_node(rstate, &node_id);
-
-       /* May happen if we accepted the node_announcement due to a local
+	struct node *node;
+	secp256k1_ecdsa_signature signature;
+	u32 timestamp;
+	struct pubkey node_id;
+	u8 rgb_color[3];
+	u8 alias[32];
+	u8 *features, *addresses;
+	struct wireaddr *wireaddrs;
+	fromwire_node_announcement(tmpctx, msg,
+	                           &signature, &features, &timestamp,
```

Comment:
Standard is to use tabs as much as possible, and spaces afterwards for finer control of indent.  So I think for this line and similar, it should have more tabs followed by 0 to 7 spaces.

Reply:
Done. In fact, there are many files where 8 spaces are used instead of tabs. They could be changed later.